src/test/java/org/openmrs/contrib/qaframework/automation/Steps.java

```diff
@@ -18,13 +19,15 @@
 
 // Use english locale, of not set, the test instance should be set to english language
 public class Steps extends ReferenceApplicationTestBase {
+	
```

Comment:
Added an empty line in reference to the artifacts collected [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/476#issuecomment-870471982)